packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import { render, screen, within } from "@testing-library/react"
+import { DateRangeValidationMessage } from "./DateRangeValidationMessage"
+
+describe("<DateRangeValidationMessage />", () => {
+  describe("will render a single validation field message", () => {
+    test("when the consumer has given it a node", () => {
+      render(
+        <DateRangeValidationMessage
+          id="error-message-id"
+          status={{
+            dateEnd: "error",
+          }}
+          validationMessage={{
+            dateEnd: <p>Validation message</p>,
```

Comment:
Do we need separate tests for string vs react component rendering?

Reply:
https://github.com/cultureamp/kaizen-design-system/pull/3277#discussion_r1118180643